lib/builderator/interface/packer.rb

```diff
@@ -49,12 +59,33 @@ def initialize(*_)
             json[:builders] << build_hash
           end
 
+          post_processors = []
+
+          # Post-processors should be considered as a sequence
+          Config.profile.current.packer.post_processor.each do |name, post_processor|
+            post_processor_hash = post_processor.to_hash
+
+            # Single, named step in a sequence
+            if post_processor_hash.empty?
+              post_processors << name
+              next
+            end
+
+            # The post-processor's type should be the same as the name
+            post_processor_hash[:type] = name
+
+            post_processors << post_processor_hash
+          end
+          json['post-processors'].push(post_processors)
```

Comment:
There's 3 different types of post-processors: a simple definition, a detailed definition, and a sequence definition.

There's no penalty for using a sequence definition (which is what we're doing) versus including logic to determine if we should use simple, detailed, or sequences.

See https://www.packer.io/docs/templates/post-processors.html for more info.